### DISCUSSION TITLE
请帮忙：续费两年，到期日未更新。

### DISCUSSION THREAD

**Discussion by yghpy** [getlantern/lantern-forum-cn#926] [General]
你好！10月2号付款续费两年，当时到期日未更新，我也没注意。今天发现我的专业版到期了，也没有激活码可用，不能使用了，麻烦帮我续上，另外请注意，当时说好赠送3个月的，感谢！
附件是我的支付宝账单，请查看。
![微信图片_20241005095948](https://github.com/user-attachments/assets/99dd5ff2-97e6-48ff-80fe-aa91ef137e70)

**Comment by yghpy** [comment]
我的邮箱
yuenli@hotmail.com

**Comment by yghpy** [comment]
[Mike777Khoo](https://github.com/Mike777Khoo)已经三天不能正常使用了。能否关注一下？

**Comment by yghpy** [comment]
请问Lantern的管理员们是集体放假了吗？

**Comment by MaxMKMax** [comment]
@yghpy 您好. 我回来了. 已经帮您提交问题上去了. 非常抱歉啊.

**Comment by yghpy** [comment]
Mike， 感谢回复。由于几天没能正常使用，所以比较着急，希望能尽快解决，请谅解。
另外，麻烦不要忘记当时说好的送三个月，谢谢！

附：论坛里有不少和我有类似问题的用户，其中不乏有等了比我更久的，大家似乎都有情绪。
个人建议，方便的话，还是公开发帖解释一下比较好，对吧？

致敬！
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年10月8日 15:13
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: yghpy ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 请帮忙：续费两年，到期日未更新。 (Discussion #926)


@yghpy<https://github.com/yghpy> 您好. 我回来了. 已经帮您提交问题上去了. 非常抱歉啊.

―

**Comment by yghpy** [comment]
[Mike777Khoo](https://github.com/Mike777Khoo)问题已解决，非常感谢帮忙。
